frontend/trip-planner/src/Components/Root.js

```diff
@@ -0,0 +1,21 @@
+import {  Outlet, NavLink } from "react-router-dom";
+import styles from './Styles/Root.css';
+
+
+function Root() {
+    return (
+        <div>
+            <div className='nav-bar'>
+                <p>The Trip Planner</p>
```

Comment:
Maybe add an icon next to the text. It'll make it look more professional. Something like this icon:

`import ModeOfTravelIcon from '@mui/icons-material/ModeOfTravel';`